### ISSUE TITLE
Buy-Widget: Wrong ID in review link aria-controls attribute

### ISSUE THREAD

**Issue by janobi** [shopware/platform#6557]
*PHP Version:* 8.3.16   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

In `storefront/component/buy-widget/buy-widget.html.twig`, the `.product-detail-reviews-link` link element still has `"review-tab-pane"` as the value for `aria-controls`, which was the ID of the element in the pre-SW6.6 template.

*_Expected behaviour:_*

Instead, it should have the ID of the new element, as referenced by the `reviewTabHref` variable in the link element's href as well.

*_How to reproduce:_*

Navigate to any product detail page with reviews, check the aria-controls attribute.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40514**



[created from NEXT-40514, comment 537577]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40514, comment 537889]

**Comment by vienthuong** [comment]
Hi @janobi,

Could you give an example what is the expected and the actual value of `aria-controls`. In my observation, the `href` and `aria-controls` have the same value, the only different is the `href` have `#` as the prefix.

**Comment by janobi** [comment]
@vienthuong The `aria-controls` attribute references the ID `review-tab-pane`, whereas the actual element has the ID `review-tab-<productId>-pane`, e.g. `review-tab-3ac014f329884b57a2cce5a29f34779c-pane`. The href as set a few lines above in the `reviewTabHref ` variable also incorporates the productId.
This is still the case with 6.6.10.2.
